src/webapp_config.py

```diff
@@ -2,10 +2,9 @@
 
 from certificates import extract_certificates, cert_dir
 from docker_helpers import docker_cp
-from service import service
 
-
-def configure_webapps(keystore_password):
+## TODO: so far as I can see this is obsolete?
```

Comment:
Yes, this is redundant - the webapps are behind nginx now, and that takes care of SSL.